src/raftstore/store/keys.rs

```diff
@@ -80,6 +80,10 @@ pub fn region_raft_prefix(region_id: u64) -> Vec<u8> {
     key
 }
 
+pub fn region_raft_prefix_len() -> usize {
+    REGION_RAFT_PREFIX_KEY.len() + mem::size_of::<u64>()
```

Comment:
but seem that if we can update raft log first, we can find raft state with the hint directly.